modelopt/torch/speculative/plugins/transformers.py

```diff
@@ -803,7 +819,8 @@ def forward(
                 past_key_values,
                 self.eagle_freeze_base_model,
                 labels,
-                kwargs,
+                precomputed_hidden_states=hidden_states,
```

Comment:
Same as above, in the current main branch  `base_model_forward()` is skipped when provided precomputed_hidden_staets. They are functionally the same but need some merge.